modules/ai/assets/js/editor/pages/form-media/views/generate/index.js

```diff
@@ -62,6 +62,9 @@ const Generate = ( { textToImageHook, predefinedPrompt = '', initialSettings = {
 	const handleSubmit = ( event ) => {
 		event.preventDefault();
 		setGenerate();
+		if ( ! settings[ IMAGE_STYLE ] ) {
+			settings[ IMAGE_STYLE ] = 'general';
+		}
```

Comment:
what is this for?

Reply:
it's a placeholder style to add additional settings in the backend